Library/Message.swift

```diff
@@ -1,12 +1,14 @@
 import CoreLocation
 import UIKit
+import SweetSwift
 
 class DateHasher {
     private var hashingDateFormatter: DateFormatter = {
         let formatter = DateFormatter()
         // Use a manually created date format.
         // Ensures strings are the same regardless of user locale.
-        formatter.dateFormat = "D MMMM yyyy"
+        formatter.dateFormat = "dd MMMM yyyy"
```

Comment:
Was using day of the year instead of day of the month. 🙈

Reply:
oopsies